Source/WebCore/style/StyleBuilderCustom.h

```diff
@@ -1656,7 +1656,7 @@ inline void BuilderCustom::applyValueFontVariantAlternates(BuilderState& builder
     }
 
     if (auto* alternatesValues = dynamicDowncast<CSSFontVariantAlternatesValue>(value)) {
-        setAlternates(alternatesValues->value());
+        setAlternates(FontVariantAlternates(alternatesValues->value()));
```

Comment:
.. and kept this code as-is.

The perf is the same and it is a little simpler.